kzz7yh-a12966/cod-ve07v1_kzz7yh-a12966.xml

```diff
@@ -167,9 +169,8 @@
    </p>
             </div>
             <div xml:id="kzz7yh-a12966-Dd1e315">
-               <head xml:id="kzz7yh-a12966-Hd1e317">Quaestio 2</head>
                <p xml:id="kzz7yh-a12966-d1e319">
-                  <lb ed="#V" n="25"/>Ex perdictis patet solutio ad aliam quaestionem: quia mens 
+                  <lb ed="#V" n="25"/>Ex predictis patet solutio ad aliam quaestionem: quia mens 
       <lb ed="#V" n="26"/>ibi accipitur pro substantia anime 
```

Comment:
I think we want to leave the question heading and add the question-title here. 

In this small paragraph, he is saying that the answer to the second question is clear from what was said in response to the first question.

So can you add `<head xml:id="kzz7yh-a12966-Hd1e317">Quaestio 2</head>` this back and add the question-title heading.